storage/engine/rocksdb/db.cc

```diff
@@ -899,9 +899,9 @@ struct IteratorGetter : public Getter {
 struct DBGetter : public Getter {
   rocksdb::DB *const rep;
   rocksdb::ReadOptions const options;
-  rocksdb::Slice const key;
+  std::string const key;
 
-  DBGetter(rocksdb::DB *const r, rocksdb::ReadOptions opts, rocksdb::Slice k)
+  DBGetter(rocksdb::DB *const r, rocksdb::ReadOptions opts, const std::string &k)
```

Comment:
Yes, it could be a slice. Even better would be if I could remember how C++11 movement semantics worked.